plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -79,115 +41,110 @@ export class EventPipelineRunner {
     // KLUDGE: This is a temporary entry point for the pipeline while we transition away from
     // hitting Postgres in the capture endpoint. Eventually the entire pipeline should
     // follow this route and we can rename it to just be `runEventPipeline`.
-    async runLightweightCaptureEndpointEventPipeline(event: PipelineEvent): Promise<EventPipelineResult> {
+    async runLightweightCaptureEndpointEventPipeline(event: PipelineEvent) {
         this.hub.statsd?.increment('kafka_queue.lightweight_capture_endpoint_event_pipeline.start', {
             pipeline: 'lightweight_capture',
         })
-        const result = await this.runPipeline('populateTeamDataStep', event)
+
+        const eventWithTeam = await this.runStep(populateTeamDataStep, this, event)
+        if (eventWithTeam != null) {
+            return await this.runEventPipeline(eventWithTeam)
```

Comment:
`runEventPipeline ` doesn't seem to be returning anything, so we're returning here to not double increment the `kafka_queue.single_event.processed_and_ingested`. This was not really obvious to comprehend 😅 

btw we're now logging the `event_pipeline.start` here too though, maybe just
```suggestion
            await this.runEventPipelineSteps(eventWithTeam)
```